.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.{js,jsx,ts,tsx,vue}]
```

Comment:
Idea: what about using `[*]` (as to target all files)?

And as this is the first of this kind in the repo, I believe we should also set `root = true` as the first line.